Bio/PDB/parse_pdb_header.py

```diff
@@ -129,18 +129,67 @@ def parse_pdb_header(infile):
     return _parse_pdb_header_list(header)
 
 
+def _parse_remark_465(line, out_dict):
+    """Parse missing residue remarks.
+
+    Fills two fields in out_dict: has_missing_residues and missing_residues.
+    It specification for REMARK 465 only gives templates, but does not say
+    they have to be followed. So we assume that not all pdb-files with a
+    REMARK 465 can be understood.
+
+    The dictionary entry "has_missing_residues" will be set to True, if
+    at least one REMARK 465 entry with non-empty text exists.
+
+    The dictionary entry 'missing_residues' will be filled with those
+    missing residues that can be parsed from the PDB header.
+    WARNING: The list out_dict['missing_residues'] will be
+    empty or incomplete, if the pdb-header is not successfully parsed.
+    """
+    if line:
+        out_dict["has_missing_residues"] = True
+        # Note that line has been stripped.
+        assert line[0] != " " and line[-1] not in "\n ", "line has to be stripped"
+    # Optional model number and residue name with 1 (e.g. for RNA) to 3 characters
+    modelnr_and_resname = "(\d+\s[\sA-Z][\sA-Z][A-Z]|[A-Z]?[A-Z]?[A-Z])"
```

Comment:
minor: regex strings should be raw strings to prevent C-style backslash expansion, e.g. `r"\b"`. (not used in this case but it's good practice to avoid confusion)